libraries/botbuilder-azure/src/cosmosDbStorage.ts

```diff
@@ -51,6 +52,10 @@ export interface CosmosDbStorageSettings {
      * @deprecated Please use [[CosmosDbPartitionedStorage]]. See https://github.com/microsoft/botframework-sdk/issues/5467
      */
     partitionKey?: string;
+    /**
+     * (Optional) http agent to use for outbound requests
```

Comment:
As to not back us into a corner with potentially weird backwards-compat/"which-write-wins" logic in the future, what's the "deprecation" story for this patch/hack?

Reply:
See above re: deprecation.